backend/app/legacy/controllers/info.py

```diff
@@ -82,3 +93,37 @@ def healthcheck() -> (Healthcheck, int):
         return healthcheck_status, 500
 
     return healthcheck_status, 200
+
+
+def sync_status() -> Tuple[SyncStatus, int]:
+    services, status = healthcheck()
+    blockchainEpoch = None
+    indexedEpoch = None
+    blockchainHeight = None
+    indexedHeight = None
+    pendingSnapshot = None
+    finalizedSnapshot = None
+    if services.blockchain == "UP":
```

Comment:
if you move the logic to the core, then you can also create an Enum class for these states, i.e.

```
class HealthcheckStates:
   DOWN = 0 
   UP = 1
```